.travis.yml

```diff
@@ -12,4 +12,5 @@ before_install:
   - docker ps -a
 
 script:
-  - curl http://localhost:8080/ -L --fail -s | grep 'Sign in</button>' -q
\ No newline at end of file
+  - curl http://localhost:8080/ -L --fail -s | grep 'Sign in</button>' -q
+  - docker-compose exec -w '/var/www/localhost/htdocs/protected' -T humhub php yii test/email 'test@example.com'
```

Comment:
I switched from travis to github actions (see .github folder) some time ago. Could you please add also add your changes to these files?
Also: Is there a way to confirm if the mail reached mailcatcher? Like some API call or something?

Reply:
I've never used GitHub actions but I can try to add it as well.

> Is there a way to confirm if the mail reached mailcatcher? Like some API call or something?

Not that I know of, but the `test/email` command will throw an error if the email cannot be delivered. So the command being successful should be enough.

If you prefer, we can add a check on the success message displayed at the end of the command:
https://github.com/humhub/humhub/blob/e904693aaf1722fd0cd71ec052b84b0c7b865e4a/protected/humhub/commands/TestController.php#L36

Reply:
@mriedmann That should be ok now with GitHub actions as well.